src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: $cardSize;
+  height: 406px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  &__img-wrapper {
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__img {
+    height: 100%;
+    width: 100%;
+    object-fit: contain;
+  }
+
+  &__product-name {   //*   это навзание моего товара
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__number-code {
+    margin-bottom: 16px;
+    color: $colorGrey;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center; //*
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 16px;
+  }
+
+  &__price-item {
+    font-size: 12px;
+    line-height: 18px;
+    color: $colorGrey;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    text-align: right;
+  }
+
+  &__buy-link {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    background-color: $color-skyBlue;
+    border: 1px solid $color-skyBlue;
+    border-radius: 5px;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: #fff;
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 12th point. Fix this in all places in your code